packages/react-instantsearch-core/src/connectors/connectInfiniteHits.js

```diff
@@ -29,6 +29,35 @@ function getCurrentRefinement(props, searchState, context) {
   return currentRefinement;
 }
 
+function getStateWithoutPage(state) {
```

Comment:
exporting helper functions
vs
expose them as parameters like `read({ state, getStateWithoutPage, isEqual }) { ... }`

What do you think of the latter?

Reply:
I wouldn't add them to the API because then it's hard to remove them since it becomes a feature.

We don't need to export them externally though, just share them internally for now.

Reply:
you're right. makes sense.

Reply:
Okay, now I remember why I didn't gather `getStateWithoutPage` in one file in the first place.

There is an in-memory cache implementation inside `connectInfiniteHits` which belongs to `-core` package.

There is a session storage implementation which is exposed as `getInfiniteHitsSessionStorageCache` from `-dom` package.

If we were to share `getStateWithoutPage`, then the place where it lives would be `-core`. Then we need to expose it so that it can be used in `-dom` package.

For a function with two lines, it looked like an overkill to me.

What do you think?

Reply:
Right, let's skip this then!